deployment/src/main/java/io/quarkiverse/tekton/client/deployment/TektonClientProcessor.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkiverse.tekton.client.deployment;
```

Comment:
```suggestion
package io.quarkiverse.tektonclient.deployment;
```
Sorry we have strict rules for package names and groupIds :)